Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -63,7 +62,7 @@ struct DashboardView: View {
                 }
                 .listStyle(.inset)
             }
-            .analyticsScreen(name: "\(Self.self)")
+            .analyticsView("\(Self.self)")
```

Comment:
Does this need to use string interpolation or can you pass `Self.self` directly?

Reply:
I'm not sure let me check